README.md

```diff
@@ -1,70 +1,141 @@
-# Getting Started with Create React App
+# 📗 Table of Contents
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+- [Hello React Front End ](#-hello-react-front-end-)
+  - [🛠 Built With ](#-built-with-)
+    - [Tech Stack ](#tech-stack-)
+    - [Key Features ](#key-features-)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites ](#prerequisites-)
+    - [Setup ](#setup-)
+    - [Install ](#install-)
+    - [Usage ](#usage-)
+  - [👥 Author ](#-author-)
+    - [👤 **Henry Odongo**](#-henry-odongo)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [📝 License ](#-license-)
 
-## Available Scripts
+<!-- PROJECT DESCRIPTION -->
 
-In the project directory, you can run:
+# 📖 Hello React Front End <a name="about-project"></a>
 
-### `npm start`
+**Hello React Front End** is a project using React and Redux for frontend to display the data coming from API [Hello-Rails-Back-End](https://github.com/Odongo006/Hello-Rails-Backend).
```

Comment:
- [ Required ] I think it would be nice if your project links to the [ Setup](https://github.com/Odongo006/Hello-Rails-Backend/tree/setub2) which has all the information regarding the `Back-end` of this application. 

## Correct link
```sh
https://github.com/Odongo006/Hello-Rails-Backend/tree/setub2
```